.github/workflows/forward-merge.yml

```diff
@@ -8,49 +8,90 @@ on:
       - prerelease/minor
 
 jobs:
-  verify-merge:
+  get-branch-names:
     # Only run if:
-    # - branch is support/master AND the last commit was a release commit
-    # - branch is prerelease/*
+    # - commit contains '[skip release]' (release commits or commits that skipped a release)
+    # - branch is prerelease/minor
     #
-    # We don't want to run on support/master for non-release commits because that would mean the
-    #   forward merge would run twice because every commit on master/support causes a new commit
-    #   pushed back to the branch. So we ignore all non-release commits on support/master because
-    #   the release workflow will commit a release commit starting with `chore: Release ...` back to
-    #   the branch and this job will run on that commit instead. Also, forward merges that skipped
-    #   a release
+    # We don't want to run a forward merge on support/master on commits that will trigger a release.
+    # This would cause forward merge to run twice (once for normal commit, and one for the release)
+    # commit. `prerelease/minor` branch does not create release commits, so those branches are safe
+    # to forward-merge. A release commit looks like "chore: Release vx.x.x [skip release]" and all
+    # commits that skip the release action also contain "[skip release]". We will run forward merge
+    # on any commit that will not trigger another commit
     if:
-      "((github.ref_name == 'support' || github.ref_name == 'master') &&
-      startsWith(github.event.head_commit.message, 'chore: Release')) || startsWith(github.ref_name,
-      'prerelease/') || contains(github.event.head_commit.message, '[skip release]')"
+      startsWith(github.ref_name, 'prerelease/') || contains(github.event.head_commit.message,
+      '[skip release]')
```

Comment:
When we added `contains(github.event.head_commit.message, '[skip release]')`, that included commits with `"chore: Release"`, so that previous requirement was dropped.